examples/nlp/asr_postprocessor/asr_postprocessor.py

```diff
@@ -60,7 +60,7 @@
 parser.add_argument("--attn_score_dropout", default=0.25, type=float)
 parser.add_argument("--attn_layer_dropout", default=0.25, type=float)
 parser.add_argument("--eval_step_frequency", default=2000, type=int)
-parser.add_argument("--data_dir", default="/dataset/", type=str)
+parser.add_argument("--data_dir", default="../../../tests/data/pred_real", type=str)
```

Comment:
Why are we using the data under test?

Reply:
it used to be there in our last release

Reply:
Where does this data come from?

Reply:
I can not find the data. Is it under here https://github.com/NVIDIA/NeMo/tree/master/tests/data?

Reply:
Yes

Reply:
In current master, there is no pred_real?

Reply:
No

Reply:
I added in this pr, please checkout the commits or changed files

Reply:
These files are just a small postion of the real data. I think these small files are created just for CI tests. You may merge this PR for now, but we should get the real data from Oleksii and share it with users.